File: query-feature-254.js
const sqlite3 = require('sql.js');

async function main() {
  const SQL = await sqlite3();
  const fs = require('fs');
  const fileBuffer = fs.readFileSync('features.db');
  const db = new SQL.Database(fileBuffer);
  const result = db.exec('SELECT * FROM features WHERE id = 254');
  if (result.length > 0) {
    const columns = result[0].columns;
    const values = result[0].values[0];
    const obj = {};
    columns.forEach((col, i) => obj[col] = values[i]);
    console.log(JSON.stringify(obj, null, 2));
  }
  db.close();
}

main().catch(console.error);

File: query_feature_254.py
import sqlite3

conn = sqlite3.connect('features.db')
c = conn.cursor()
c.execute('SELECT * FROM features WHERE id = 254')
row = c.fetchone()
cols = [d[0] for d in c.description]
result = dict(zip(cols, row))
for k, v in result.items():
    print(f"{k}: {v}")
conn.close()
